lib/pf/services/manager/collectd.pm

```diff
@@ -58,7 +61,10 @@ sub generateCollectd {
     $tags{'graphite_host'} = "$Config{'monitoring'}{'graphite_host'}";
     $tags{'graphite_port'} = "$Config{'monitoring'}{'graphite_port'}";
     $tags{'hostname'}      = hostname;
-    $tags{'db_host'}       = "$Config{'database'}{'host'}";
+    $tags{'db_host'}
+        = $cluster_enabled
+        ? $management_network->tag('ip')
```

Comment:
But we should be going through haproxy to find the server that is currently alive.

If we're on the server that isn't the one offering DB service, that won't connect to a valid MySQL DB

So this is already set to 127.0.0.1 (database.host) which should be the right configuration